tools/lint.sh

```diff
@@ -4,9 +4,32 @@
 
 set -e  # exit on error
 
-if [[ ! -z "$1" ]]; then
-    cd "$1"
-fi
+CHECK=""
+
+while [ "$1" != "" ]; do
+	case "$1" in
+		--check)
+		  CHECK="--check";;
```

Comment:
Maybe this is meant to be indented 8 spaces? Not sure

Reply:
Looks like a copy pasta fail.  Those are tab characters that got pulled in. 
I normalized the indent to 2 spaces as that's what was there previously with the for loop at the bottom.